doc/tools/docgram/common.edit_mlg

```diff
@@ -2491,6 +2497,7 @@ RENAME: [
 | field_body field_spec
 | field_def field_val
 | bindings_with_parameters alias_definition
+| rewstrategy_prefix_op rewstrategy
```

Comment:
```suggestion
```

Reply:
Why do you suggest this removal?

Reply:
I think it's confusing to have the documented `rewstrategy` mean a single thing in the sequence, when the scope in tactic notations and Ltac2 notations would mean a sequence of them.  Even if this makes the grammar a bit more verbose, I think it's better to have `rewstrategy` be `LIST1 ";" rewstrategy1` or whatever